.classpath

```diff
@@ -53,8 +53,7 @@
 	<classpathentry kind="lib" path="/jars/lib/jars/mysql/mysql-connector-java-5.1.22-bin.jar"/>
```

Comment:
Needs:
version bump to 0.9.1 (in the server class and docsource/conf.py)
release notes (create a new 0.9.1 version in the notes) (see ./docsource)
documentation updates since the deploy configuration is different (see ./docsource)
update to deploy.cfg.example
update to deploy template in deployment dir